src/locales/en.json

```diff
@@ -12,6 +12,4 @@
 	},
 	"MESSAGES": {
 		"HEALTHCHECK": "OK"
-	},
-	"Cast to string failed for value \"{ field": " 'title' }\" at path \"title\" for model \"TodoItem\""
 }
```

Comment:
Please add new line at EOF. You might want to take a look at EditorConfig (https://editorconfig.org/) which should take care of this, as well as make sure indentation is consistent.